app/views/file_not_available.scala.html

```diff
@@ -15,22 +15,27 @@
  *@
 
 @import config.ApplicationConfig
+@this(
+        govuk_wrapper: views.html.govuk_wrapper,
+        govukBackLink: GovukBackLink
+)
 @()(implicit request: Request[_], messages: Messages, appConfig: ApplicationConfig)
-	@ras_main(title = messages("file.not.available.page.title")) {
+	@govuk_wrapper(pageTitle = messages("file.not.available.page.title")) {
 
-    <a class="link-back" id="back" href="@routes.ChooseAnOptionController.get" data-journey-click="navigation - link:File not available:Back">@messages("back")</a>
+    @govukBackLink(BackLink(
+        href = routes.ChooseAnOptionController.get.url,
+        content = Text(messages("back")),
+        attributes = Map("id" -> "back", "class" -> "link-back", "data-journey-click" -> "navigation - link:File not available:Back")
+    ))
 
     <div class="row">
```

Comment:
The class "row" could be removed from these div elements based on the comment further up for consistancy.